.github/workflows/build.yml

```diff
@@ -3,48 +3,36 @@ name: Build + Test
 on: [push, pull_request]
 
 jobs:
-  test:
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
-        os: [windows-latest, ubuntu-latest]
-        node-version: [16.x]
+  build:
+    runs-on: [ubuntu-latest]
 
-    name: ${{ matrix.os }} (Node.js ${{ matrix.node-version }})
+    name: Build Material Icon Theme
 
     steps:
       - name: Checkout 🛎
         uses: actions/checkout@v4
 
-      - name: Setup Node.js ${{ matrix.node-version }} ⚙
-        uses: actions/setup-node@v3
+      - name: Setup Platform 🛠
+        uses: oven-sh/setup-bun@v1
         with:
-          node-version: ${{ matrix.node-version }}
+          bun-version: latest
 
-      - name: Run display server 🖥
-        run: /usr/bin/Xvfb :99 -screen 0 1024x768x24 > /dev/null 2>&1 & echo "Started xvfb"
-        shell: bash
-        if: ${{ success() && matrix.os == 'ubuntu-latest' }}
+      - name: Install dependencies 📦
+        run: bun install --frozen-lockfile
 
-      - name: Install node_modules 📦
-        run: npm ci
-
-      - name: Cache node_modules 💾
-        uses: actions/cache@v3
+      - name: Cache dependencies 💾
+        uses: actions/cache@v4
```

Comment:
I've personally found this to take more time than just letting Bun download them freshly, and so has the Bun team: https://github.com/oven-sh/setup-bun/issues/14#issuecomment-1714116221

Reply:
I've removed it and it's faster and easier without the cache. Thank you for that information 😊

Reply:
> Thank you for that information 😊

You're welcome! Thanks for linking PRs in the changelog 😉

Reply:
You're welcome 🤗 Linking the PRs in the change log is done automatically btw, so I'll forward your thanksgiving to the pipeline 😉